### PR TITLE
Add target="_blank" for github and twitter links

### PR DESCRIPTION
**Pull request by brunoerg**
When the user clicks on a link (github or twitter), I think the best solution is opening a new tab instead of loading it in the same tab.

### CODE REVIEW

File: includes/template.pug [hunk 1]
```diff
@@ -18,9 +18,9 @@ mixin donatee(d)
       h3.name
         a(href=`/${d.id}/`)= d.name
       if d.twitter
-        a.twitter(href=`https://twitter.com/${d.twitter}`) Twitter
+        a.twitter(href=`https://twitter.com/${d.twitter}`, target="_blank") Twitter
```
Comment (dennisreimann) [inline includes/template.pug:21]:
```suggestion
        a.twitter(href=`https://twitter.com/${d.twitter}` target="_blank") Twitter
```

File: includes/template.pug [hunk 2]
```diff
@@ -18,9 +18,9 @@ mixin donatee(d)
...
       if d.github
-        a.github(href=`https://github.com/${d.github}`) GitHub
+        a.github(href=`https://github.com/${d.github}`, target="_blank") GitHub
```
Comment (dennisreimann) [inline includes/template.pug:23]:
```suggestion
        a.github(href=`https://github.com/${d.github}` target="_blank") GitHub
```

**Review by dennisreimann — CHANGES_REQUESTED** [review]
Good point, thanks for the contribution! Let's remove the attribute commas, then this is good to go 👍

File: includes/template.pug (hunk 2 above)
Comment (brunoerg) [reply includes/template.pug:23]: Great!

File: includes/template.pug (hunk 1 above)
Comment (brunoerg) [reply includes/template.pug]: Fixed

File: includes/template.pug (hunk 2 above)
Comment (brunoerg) [reply includes/template.pug]: Fixed

**Review by dennisreimann — APPROVED** [review]

**Pull request merged**